js/src/redux/selectors/fallbacks.js

```diff
@@ -43,11 +44,21 @@ export const getContentImage = state => {
  * @returns {string} The sidewide image url.
  */
 export const getImageFallback = state => {
-	const featuredImage = get( state, "snippetEditor.data.snippetPreviewImageURL", "" );
-	const contentImage = get( state, "settings.socialPreviews.contentImage", "" );
-	const siteWideImage = get( window.wpseoScriptData, "metabox.showSocial.facebook" ) && get( state, "settings.socialPreviews.sitewideImage", "" );
+	const fallbacks = [
+		{ featuredImage: get( state, "snippetEditor.data.snippetPreviewImageURL", "" ) },
+		{ contentImage: get( state, "settings.socialPreviews.contentImage", "" ) },
+		{ siteWideImage: get( window.wpseoScriptData, "metabox.showSocial.facebook" ) && get( state, "settings.socialPreviews.sitewideImage", "" ) },
+	];
```

Comment:
I see this formatting has been used in the woocommerce-seo PR
```
		if ( productGalleryFallbackImage ) {
			fallbacks.push( { productGalleryImage: productGalleryFallbackImage } );
		}
```
I believe the intention here is to create a stable order with identifiable entries, so that add-ons can position extra fallbacks relative to the other entries.

But I'm unconvinced these single depth, single key-value objects provide the best solution. They require (in my opinion) a lot of `Object.values()` and potentially `Object.keys()`. As seen below on line 55.

Can we not simply add these image URL's in an array, and then use `find()` to find the first truthy entry? I don't think the current object form adds any advantage other than potentially  being able to say "add my image behind the "contentImage"", and even then you can't be sure there isn't another add-on trying to do the same.